Readme.md

```diff
@@ -22,3 +22,14 @@ make
 The test array builder will create a folder called `arrays` in the current working directory of where it is run.
 In the arrays folder will be a sub folder with the tiledb version. This can be copied to the top level directory
 and it will be used by TileDB for unit testing.
+
+## Reviewers
+Adding a review for a PR here it's not trivial, opening a PR in the browser doesn't work (usually a PR will add a new array and it's 5k+ files),
+cloning takes forever, we also don't have an older branch which has less data in it so you can clone that directly, here's one way to do add a review:
```

Comment:
This sentence isn't super readable as-is. I'd suggest splitting it into a few sentences. Something like this:
```suggestion
Reviewing PRs in this repository is not always trivial. Many PRs add 5k+ files which cannot be opened in the browser, and the repository itself takes a long time to clone. Additionally, there is no alternative, older branch with less data to be cloned. To overcome these hurdles, here is one suggested way to add a review:
```